### PR TITLE
Update guide to Open JDK 12

### PR DESCRIPTION
**Pull request by manishtr**
### Change Description ###

Update guide to Open JDK 12.

Signed-off-by: manish.r <manish.r@samsung.com>

### CODE REVIEW

**Comment by manishtr** [comment]
> Hi,
> In index.md,overview.md,install.md, I can stillsee the "OpenJDK 10" comment.
> Could you change it to OpenJDK 12?
> Please check the three files,again.

Fixed

File: docs/application/vstools/install.md [hunk 1]
```diff
@@ -22,12 +22,14 @@ To work with Visual Studio Tools for Tizen, your computer must have:
...
 
-  JDK 8 is needed to use the Tizen Baseline SDK. OpenJDK is not supported.
+  You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 12 to use 
+Tizen Baseline SDK. Make sure you download and install the EXACT version.
 
-  Download the JDK 8 from the official [Oracle Web site](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Select the appropriate platform for your hardware architecture and Windows&reg; version. Run the downloaded executable file and follow the displayed instructions.
-  - JDK 9 will be supported soon.
+  - [Oracle Java Development Kit(JDK) 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+ 
+  - OpenJDK 12 and OpenJFX: [OpenJDK 12 and OpenJFX Installation Guide](https://developer.tizen.org/development/articles/openjdk-12-and-openjfx-installation-guide#install-openjdk-for-windows).
```
Comment (journey2w) [inline docs/application/vstools/install.md]: We don't use TD URL to link pages except there are no pages in tizen-docs repo.

Comment (manishtr) [reply docs/application/vstools/install.md]: Fixed, add the link to openjdk.md file.
Thanks for your suggestion

**Comment by journey2w** [comment]
TD update was done.

**Pull request merged**